src/main/java/org/jboss/xavier/analytics/pojo/output/WorkloadInventoryReportModel.java

```diff
@@ -4,40 +4,49 @@
 import org.hibernate.annotations.GenericGenerator;
 import org.hibernate.annotations.Parameter;
 
+import javax.persistence.CollectionTable;
 import javax.persistence.ElementCollection;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.Index;
 import javax.persistence.JoinColumn;
 import javax.persistence.OneToOne;
+import javax.persistence.Table;
 import javax.transaction.Transactional;
 import java.math.BigDecimal;
-import java.util.ArrayList;
 import java.util.Date;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Set;
 
 @Entity
 @Transactional
+@Table(
+        indexes = {
+                @Index(name = "WorkloadInventoryReportModel_" +
+                        WorkloadInventoryReportModel.ANALYSIS_ID + "_index",
+                        columnList = WorkloadInventoryReportModel.ANALYSIS_ID, unique = false)
```

Comment:
`unique` value is by default `false` so no need to write it again.

Reply:
As above